mac/deploy_mac.sh

```diff
@@ -67,18 +67,13 @@ build_app()
 
 build_installer_image()
 {
-    # Install dmgbuild (for the current user), this is required to build the installer image
-    python -m ensurepip --user --default-pip
-    python -m pip install --user dmgbuild==1.4.2
-    local dmgbuild_bin="$(python -c 'import site; print(site.USER_BASE)')/bin/dmgbuild"
-
+    # this needs brew to be installed first
+    # install create-dmg
+    brew install create-dmg
     # Get Jamulus version
     local app_version="$(cat "${project_path}" | sed -nE 's/^VERSION *= *(.*)$/\1/p')"
-
     # Build installer image
-    "${dmgbuild_bin}" -s "${macdeploy_path}/deployment_settings.py" -D background="${resources_path}/installerbackground.png" \
-        -D app_path="${deploy_path}/$1.app" -D server_path="${deploy_path}/$2.app" \
-        -D license="${root_path}/COPYING" "$1 Installer" "${deploy_path}/$1-${app_version}-installer-mac.dmg"
+    create-dmg --volname "$1 Installer" --background "${resources_path}/installerbackground.png" --window-pos 200 120 --window-size 800 400 --app-drop-link 600 185 --skip-jenkins --eula "${root_path}/COPYING" "${deploy_path}/$1-${app_version}-installer-mac.dmg" "${deploy_path}/"
```

Comment:
Could you wrap this line nicely, so Github diff doesn't scroll sideways, please...  (clang-format for shell scripts needed... mmm...)

Reply:
Done. I hope it still works (there were errors in the script as I re-ordered the arguments)